xiplot/plugin.py

```diff
@@ -139,6 +139,68 @@ def get_plugin_filepaths(dir_path=""):
         return []
 
 
+def install_local_plugin(plugin_path: str):
+    plugin_path = str(Path(plugin_path).resolve())
+
+    try:
+        import asyncio
+
+        import micropip
+
+        # FIXME: get_running_loop doesn't work in WASM
+        # Needs to run in the background and report back via hidden UI
+        asyncio.get_running_loop().run_until_complete(
+            micropip.install(f"emfs:{plugin_path}")
+        )
```

Comment:
I have tried to get this approach to work, but did not manage to find a way (that is why `scikit-learn` is lazily loaded using javascript and `setTimeout` in the WASM branch)

Reply:
Yeah, I have an idea for solving it, similar to how the clustering are done. In this case, launch async install with a callback that interacts with a hidden input in the page and tells us about the success or failure which then again triggers a callback for the UI. Ugly, but it should work